crates/ironrdp-bench/benches/bench.rs

```diff
@@ -10,8 +10,8 @@ pub fn rfx_enc_tile_bench(c: &mut Criterion) {
     let quant = rfx::Quant::default();
     let algo = rfx::EntropyAlgorithm::Rlgr3;
     let bitmap = BitmapUpdate {
-        top: 0,
-        left: 0,
+        x: 0,
+        y: 0,
```

Comment:
question: I think I’m fine with this change if you really want to, but I’m not sure whether this is really more idiomatic and less confusing. For instance, `x` and `y` could denote the center of the rectangle, while `top` and `left` and much more unambiguous.

Reply:
To me, "top/left" do not clearly denote coordinates. It also bothers me a bit to have "foo.x + bar.left" for example. I would rather use the same field name. But if we rather keep mixing the two, fine.

Reply:
Okay, let’s go with x, y